src/ert/scheduler/lsf_driver.py

```diff
@@ -444,6 +449,24 @@ async def _get_exit_code_from_bhist(self, job_id: str) -> int:
 
         return LSF_FAILED_JOB
 
+    async def _dump_bhist_job_summary_to_runpath(self, job_id: str) -> None:
+        runpath = self._jobs[job_id].runpath
+        job_name = self._jobs[job_id].name
+        file_path = runpath / f"{job_name}.LSF-bhist-l"
+        bhist_with_args: list[str] = [
+            str(self._bhist_cmd),
+            "-l",  # long format
+            str(job_id),
```

Comment:
This is already a str according to typing in the function definition

Reply:
I think we tested this one with @jonathan-eq and it only worked with `str(job_id)` :shrug:

Reply:
Hmm, covering up another bug?

Reply:
```
Experiment failed with the following error: Experiment failed! All realizations failed!
Exception in scheduler task poll_task: expected string or bytes-like object
Traceback: Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/private/jonak/ert/src/_ert/async_utils.py", line 53, in _done_callback
    raise exc
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 371, in poll
    await self._process_job_update(job_id, job)
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 414, in _process_job_update
    await self._dump_bhist_job_summary_to_runpath(job_id)
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 461, in _dump_bhist_job_summary_to_runpath
    process_success, process_message = await self._execute_with_retry(
  File "/private/jonak/ert/src/ert/scheduler/driver.py", line 95, in _execute_with_retry
    f'Command "{shlex.join(cmd_with_args)}" succeeded with {outputs}'
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 316, in join
    return ' '.join(quote(arg) for arg in split_command)
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 316, in <genexpr>
    return ' '.join(quote(arg) for arg in split_command)
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 325, in quote
    if _find_unsafe(s) is None:
TypeError: expected string or bytes-like object

Experiment failed! All realizations failed!
Exception in scheduler task poll_task: expected string or bytes-like object
Traceback: Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/private/jonak/ert/src/_ert/async_utils.py", line 53, in _done_callback
    raise exc
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 371, in poll
    await self._process_job_update(job_id, job)
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 414, in _process_job_update
    await self._dump_bhist_job_summary_to_runpath(job_id)
  File "/private/jonak/ert/src/ert/scheduler/lsf_driver.py", line 461, in _dump_bhist_job_summary_to_runpath
    process_success, process_message = await self._execute_with_retry(
  File "/private/jonak/ert/src/ert/scheduler/driver.py", line 95, in _execute_with_retry
    f'Command "{shlex.join(cmd_with_args)}" succeeded with {outputs}'
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 316, in join
    return ' '.join(quote(arg) for arg in split_command)
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 316, in <genexpr>
    return ' '.join(quote(arg) for arg in split_command)
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/shlex.py", line 325, in quote
    if _find_unsafe(s) is None:
TypeError: expected string or bytes-like object
```

Reply:
Apparently this was caused by passing the path object of the bhist_command, not related to  job_id at all.